bootcamp/tutorials/quickstart/hybrid_search_with_milvus.ipynb

```diff
@@ -0,0 +1,552 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.    # Create indices for the vector fields
</pre><p># <strong>To make vector search efficient,</strong> we need to create indices for the vector fields</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1389/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>